packages/expansion-hub/src/RevHub.ts

```diff
@@ -2,4 +2,6 @@ import {RevHubType} from "./RevHubType";
 
 export interface RevHub {
     type: RevHubType
+
+    on(eventName: "error", listener: (...args: any[]) => void): RevHub
```

Comment:
```suggestion
    on(eventName: "error", listener: (error: Error) => void): RevHub
```

Let's guarantee that we're sending them an `Error` object. If somehow something other than an error gets thrown, we can call `toString()` on it and wrap the result in an `Error`.

This also applies to `ExpansionHub`, of course.

Reply:
Updated the args. We should also make a type-safe method for emitting to make sure we can guarantee that all emitted errors have a single parameter of type Error.